pyfranca/franca_lexer.py

```diff
@@ -73,6 +75,10 @@ class Lexer(object):
         "ID",
         "INTEGER",
         "FILE_NAME",
+        "DOUBLE_VAL",
```

Comment:
it is necessary, using token double conflicts with keyword double. 

tokens = **[keyword.upper() for keyword in keywords]** + [
        "ID",
        "INTEGER",
        "FILE_NAME",
        "DOUBLE_VAL",
        "FLOAT_VAL",
        "STRING_VAL",
        "BOOLEAN_VAL"
    ]